nearcore/src/lib.rs

```diff
@@ -114,9 +120,19 @@ fn create_db_checkpoint(path: &Path, near_config: &NearConfig) -> Result<PathBuf
     Ok(checkpoint_path)
 }
 
+fn get_store_version_with_config(path: &Path, near_config: &NearConfig) -> u32 {
```

Comment:
The function will open and then immediately close the database. I honestly don’t think we need to pass configuration from NearConfig here.

Reply:
My thought was that in the future there could be other store parameters REQUIRED for db to correctly open even for just reading the version so it's better to open the store with the same parameters everywhere.

Reply:
I think we can worry about required options when we actually have any. For now I wouldn’t complicate it.